### PR TITLE
CNV 19687: Wizard sidebar editor other tabs

### PR DESCRIPTION
**Pull request by upalatucci**
## 📝 Description

Sidebar editor for wizard tabs: schedule, metadata, networks ,disks

## 🎥 Demo

Screenshot from 2022-09-21 15-09-32
Screenshot from 2022-09-21 15-09-29
Screenshot from 2022-09-21 15-09-26
Screenshot from 2022-09-21 15-09-23
Screenshot from 2022-09-21 15-09-20

### CODE REVIEW

File: src/views/catalog/wizard/tabs/metadata/wizard-metadata-tab.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.wizard-metadata-tab {
+
+    .wizard-description-item__title {
+        justify-content: space-between;
+        flex: 1;
+      }
+
+    &__description-list {
+        max-width: 700px;
+        margin-top: 1rem;
```
Comment (metalice) [inline src/views/catalog/wizard/tabs/metadata/wizard-metadata-tab.scss:10]: var?

File: src/views/catalog/wizard/tabs/scheduling/wizard-scheduling-tab.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+.wizard-scheduling-tab {
+    &__grid {
+      margin-top: 1rem;
```
Comment (metalice) [inline src/views/catalog/wizard/tabs/scheduling/wizard-scheduling-tab.scss:4]: var?

File: src/views/catalog/wizard/tabs/scripts/WizardScriptsTab.scss [hunk 3]
```diff
@@ -1,8 +1,16 @@
...
   }
-  .pf-c-expandable-section__toggle-text {
-    font-size: 18px;
+  
+  &-expanable-section {
+    .pf-c-expandable-section__toggle {
+      align-items: center;
+    }
+    .pf-c-expandable-section__toggle-text {
+      font-size: 18px;
```
Comment (metalice) [inline src/views/catalog/wizard/tabs/scripts/WizardScriptsTab.scss:13]: vars?

**Comment by upalatucci** [comment]
all these vars fixed!! aaaah :smile:

**Pull request merged**